src/sentry/static/sentry/app/views/organizationIntegrations/migrationWarnings.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {map} from 'lodash';
+
+import {t} from 'app/locale';
+import AddIntegration from 'app/views/organizationIntegrations/addIntegration';
+import AlertLink from 'app/components/alertLink';
+
+export default class MigrationWarnings extends React.Component {
+  static propTypes = {
+    unmigratableRepos: PropTypes.object.isRequired,
+    providers: PropTypes.array.isRequired,
+    onInstall: PropTypes.func.isRequired,
+  };
+
+  render() {
+    return map(this.props.unmigratableRepos, (repos, orgName) => {
+      // Repos use 'visualstudio', Integrations use 'vsts'. Normalize to 'vsts'.
+      const key = repos[0].provider.id == 'visualstudio' ? 'vsts' : repos[0].provider.id;
+
+      const provider = this.props.providers.find(p => p.key == key);
+
+      return (
+        <AddIntegration
+          key={provider.key}
+          provider={provider}
+          onInstall={this.props.onInstall}
+        >
+          {onClick => (
+            <AlertLink onClick={onClick} href="#">
+              {t(
+                `Your ${orgName} repositories can't send commit data to Sentry.` +
+                  `Add a ${orgName} ${provider.name} instance here.`
+              )}
```

Comment:
Could you switch this out to a parameterized translation? 

```jsx
tct('Your [orgName] repositories can't send commit data to Sentry. Add a [orgName] [providerName] instance here.', {
  orgName,
  providerName: provider.name,
})
```

Reply:
Honestly wasn't sure how `tct` worked and couldn't discern from the code/not-really-existing-docs.

Reply:
Yeah there's absolutely no docs for it unfortunately